ci/run_black.ps1

```diff
@@ -0,0 +1,19 @@
+$ExitCode = 0
+$GitDiff = git diff origin/master
+# If there is no diff between master, then black everything.
+if ( $GitDiff -eq $null ) {
+    black .
```

Comment:
We don't want to actually modify the files here, we just want to check that they don't need blackened. I think black has a `--diff` and `--check` flags that might do this without actually modifying the file. And be sure that black should return an exit code of 0 if it passed (meaning no changes were necessary) and a non-zero exit code if it needs changes.